EIPS/eip-1.md

```diff
@@ -42,19 +42,23 @@ The EIP process begins with a new idea for Ethereum. It is highly recommended th
 
-Once the champion has asked the Ethereum community whether an idea has any chance of acceptance a draft EIP should be presented as a [pull request]. This gives the author a chance to continuously edit the draft EIP for proper formatting and quality. This also allows for further public comment and the author of the EIP to address concerns about the proposal.
+An automated process will check all pull requests; if the request passes the format checks and contains only additions or edits to drafts owned by the submitter, it will be automatically merged. No editorial control is exercised over the content of an EIP draft.
 
-If the EIP collaborators approve, the EIP editor will assign the EIP a number (generally the issue or PR number related to the EIP), label it as Standards Track, Informational, or Meta, give it status “Draft”, and add it to the git repository. The EIP editor will not unreasonably deny an EIP. Reasons for denying EIP status include duplication of effort, being technically unsound, not providing proper motivation or addressing backwards compatibility, or not in keeping with the Ethereum philosophy.
+Standards track EIPs of type 'Core' are updated from 'Draft' to 'Accepted' status when a core devs meeting has accepted the draft and the participants have announced their intention to implement the EIP for inclusion in a future hard fork. At that point, an EIP number is assigned by the editors and the EIP is moved to the EIPS directory and renamed accordingly. Once a Core EIP has been implemented in at least three clients, and all pass a common set of test suites, the status of the EIP is updated to 'Final'. An EIP can also be assigned status “Deferred”. The EIP author or editor can assign the EIP this status when no progress is being made on the EIP.
 
-Standards Track EIPs consist of three parts, a design document, implementation, and finally if warranted an update to the [formal specification]. The EIP should be reviewed and accepted before an implementation is begun, unless an implementation will aid people in studying the EIP. Standards Track EIPs must be implemented in at least three viable Ethereum clients before it can be considered Final.
+Standards track, non-core EIPs remain in Draft status until at least one implementation has been completed and referenced in the Implementations section of the draft. At that point, the draft's author may apply to the EIP editor, by way of a pull request, to assign a number to the EIP and update its status to 'Final', moving it to the EIPs directory and renaming it accordingly.
 
```

Comment:
Are we missing the definition of how the Standards track, non-core EIPs go through the Accepted/Rejected/Withdrawn stage as defined here https://github.com/ethereum/EIPs/blob/master/EIPS/eip-1/process.png ?

Reply:
Non-core EIPs don't have an "Accepted" state; they go straight from Draft to Final. The diagram will need updating accordingly.

Reply:
This is too big a jump to be based off 1 implementation (mostly done by the eip author) and EIP editors' discretion (as much as we trust them). 
This relates to what @fulldecent has been saying here and in https://ethereum-magicians.org/t/suggestion-to-have-a-community-feedback-period-for-each-eip/47/7 about the paramount task of keeping track of Draft EIPs and the ease with which a draft can go under one's radar and slip in without you noticing (unless you have invested the said 100 hours to read them all). This is something that keeps a few of us up at night it seems.

Reply:
One suggestion would be an additional prerequisite for moving to "Final" to have at least one other referenced implementation in the community.

Reply:
Isn't that what this sentence says? Or are you suggesting it should require 2+ implementations?

Reply:
I was suggesting perhaps at least one more implementation (making them 2+ in total) than the one you are proposing which is likely going to be completed by the author themselves.
After all the decision on whether something becomes a de-facto standard lies more with how wide is its adoption.

Reply:
Just an idea of maybe tying this with `ethpm` to track adoption even though progress there feels stuck solid and not moving for months now?

Reply:
Seems reasonable, since standards are most useful with multiple implementations. What about something where there's one good implementation used by multiple projects, though? Does that count as one, or several?

What do others think?